rts/Sim/Misc/QuadField.cpp

```diff
@@ -328,7 +328,7 @@ void CQuadField::GetQuadsOnRay(QuadFieldQuery& qfq, const float3& start, const f
 	}
 }
 
-
+#ifndef UNIT_TEST
```

Comment:
If this isn't just a temporary bandage then consider
```suggestion
#ifndef UNIT_TEST // exempted for reason XYZ
```

Reply:
I couldn't figure out the root cause of issue with this function from the first attempt.